app/models.py

```diff
@@ -129,12 +129,25 @@ def validates_framework(self, key, framework):
 
     slug_pattern = re.compile("^[\w-]+$")
 
-    @validates("slug")
+    @validates('slug')
     def validates_slug(self, key, slug):
         if not self.slug_pattern.match(slug):
             raise ValidationError("Invalid slug value '{}'".format(slug))
         return slug
 
+    @validates('clarification_questions_open')
+    def validates_clarification_questions_open(self, key, value):
+        # Note that because of a race condition between this key and `status`, it _is_ possible to be in a state where
```

Comment:
It's not technically a race condition - it all happens in one thread of execution, it's just the `__init__` doesn't know it should set field `A` before field `B` or vice versa.